src/fastserve/models/__main__.py

```diff
@@ -76,6 +76,7 @@
 elif args.model == "huggingface":
     app = ServeHuggingFace(
         model_name=args.model_name,
+        use_gpu=True if args.use_gpu else False,
```

Comment:
how about we accept `device="cuda"` like [here](https://github.com/gradsflow/fastserve-ai/blob/c200dbcb17a86047a13b709777d5162397579e47/src/fastserve/models/ssd.py#L19C42-L19C48)?

Reply:
make sense